lib/app/features/chat/messages/views/components/message_types/emoji_message/emoji_message.dart

```diff
@@ -4,11 +4,19 @@ import 'package:flutter/material.dart';
 import 'package:ion/app/extensions/extensions.dart';
 import 'package:ion/app/features/chat/messages/views/components/message_item_wrapper/message_item_wrapper.dart';
 import 'package:ion/app/features/chat/messages/views/components/message_metadata/message_metadata.dart';
+import 'package:ion/app/features/chat/messages/views/components/message_reactions/message_reactions.dart';
+import 'package:ion/app/features/chat/model/message_reaction_group.dart';
 
 class EmojiMessage extends StatelessWidget {
-  const EmojiMessage({required this.emoji, required this.isMe, super.key});
+  const EmojiMessage({
+    required this.emoji,
+    required this.isMe,
+    this.reactions,
+    super.key,
+  });
   final bool isMe;
```

Comment:
@ice-hector  thanks for suggestion 👍  i ll do it with different PR because it affects all message types and i have to create providers for chat. now it works with static texts/images etc.